package.json

```diff
@@ -132,7 +132,8 @@
     "test:karma:test-utils": "cross-env PERFORMANCE=false COVERAGE=false BABEL_NO_MODULES=true karma start karma.conf.js --grep=test-utils/test/shared/**.js --no-single-run",
     "test:karma:bench": "cross-env PERFORMANCE=true COVERAGE=false BABEL_NO_MODULES=true karma start karma.conf.js --grep=test/benchmarks/**.js --single-run",
     "benchmark": "npm run test:karma:bench -- no-single-run",
-    "lint": "run-s eslint",
+    "lint": "run-s eslint tsc",
+    "tsc": "tsc -p jsconfig-lint.json",
```

Comment:
Let's add this to CI now that it passes so we can keep that

Reply:
Yup! It now runs as part of the `lint` command: https://github.com/preactjs/preact/actions/runs/6664355219/job/18111803276?pr=4173#step:5:21